ixo/chain.json

```diff
@@ -3,9 +3,35 @@
     "chain_name": "ixo",
```

Comment:
Should the chain name also be `impacthub` then?

Reply:
Yes you are correct